nodeprobe/nodeprobe.go

```diff
@@ -109,7 +110,7 @@ func (p *Prober) probe(ctx context.Context) {
 	p.healthy.Store(healthy.Load())
 
 	if !p.healthy.Load() {
-		p.logger.Error("not all nodes are healthy")
+		p.logger.Error("not all Ethereum nodes are healthy")
```

Comment:
While I agree that "Ethereum nodes" is clearer, we also add the event syncer to the list of nodes, and it doesn't belong to eth nodes

Reply:
adjusted log line to list all 3:
```go
		p.logger.Error(fmt.Sprintf(
			"not all nodes (%s, %s, %s), are healthy",
			EventSyncerNode,
			ConsensusClientNode,
			ExecutionClientNode,
		))
```